protected/views/site/faq.php

```diff
@@ -42,7 +42,7 @@
                     </div>
                     <div id="panel02" aria-labelledby="heading02" class="panel-collapse collapse">
                         <div class="panel-body">
-                            <p>At the present only GigaScience.</p>
+                            <p>Currently GigaDB work very closly with GigaScience and GigaByte journals. If you have data relating to a publication in another journal you may wish to consider splitting the paper into a research paper and a datanote, the later can be submitted it to GigaByte and we can then host your data in GigaDB which can be referenced in any research article using those data.</p>
```

Comment:
There are some typos in this answer by the curator. Here is a corrected answer:

Currently, GigaDB works very closely with GigaScience and GigaByte journals. If you have data relating to a publication in another journal, you may wish to consider splitting the paper into a research paper and a data note; the latter can be submitted to GigaByte then we can host your data in GigaDB which can be referenced in any research article using those data.